content/development/contribution_guidelines.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Contribution Guidelines"
+#menuTitle: ""
+chapter: false
+weight: 5
+---
+
+> The CRS project values third party contributions. To make the contribution process as easy as possible, a helpful set of contribution guidelines are in place which all contributors and developers are asked to adhere to.
+
+## Getting Started with a New Contribution
+
+1. Sign in to [GitHub](https://github.com/join).
+2. Open a [new issue](https://github.com/coreruleset/coreruleset/issues) for the contribution, *assuming a similar issue doesn't already exist*.
+    * **Clearly describe the issue**, including steps to reproduce if reporting a bug.
+    * **Specify the CRS version in question** if reporting a bug.
+    * Bonus points for submitting tests along with the issue.
+3. Fork the repository on GitHub and begin making changes there.
```

Comment:
```suggestion
3. Fork the repository on GitHub and begin making changes there.
4. We prefer signed commits. If you don't sign your commits yet please take a look at the [GitHub documentation](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).
```

Reply:
Suggestion added (in a reference material style tone, i.e. without pronouns.)